OpenTabletDriver/Vendors/Wacom/IntuosV1/IntuosV1ReportParser.cs

```diff
@@ -22,6 +22,10 @@ private IDeviceReport GetToolReport(byte[] report)
         {
             if (report[0] == 0x10 && report[1] == 0x20)
                 return new DeviceReport(report);
+            if (report[1] == 0xE8 || report[1] == 0xA8)
+                return new IntuosV1MouseReport(report);
+            if (report[1] == 0xEA || report[1] == 0xAA) // ignore secondary mouse reports
+                return null;
```

Comment:
See if this works better.
```suggestion
            if (((report[1] >> 1) & 0x0F) == 0x04)
                return new IntuosV1MouseReport(report);
```

Reply:
Discussed this on Discord the other day, but figured I'd note the same things here for the sake of organization.

This parser change works aside from the mouse sometimes immediately sending and holding whatever left click is bound to when it enters range of the tablet, particularly from the edges of the tablet area. From the Discord conversation, it sounded as if this was a known issue with all tablet tools and is something that is beyond the scope of this particular parser, so if that is confirmed to be the case then this change can be committed.